system/jhtree/jhtree.cpp

```diff
@@ -1856,8 +1847,9 @@ bool CKeyCursor::_lookup(bool exact, unsigned lastSeg, bool unfiltered, KeyStats
         }
         if (!eof)
         {
+            verifyex(node && node->getKeyAt(nodeKey, recordBuffer));
```

Comment:
not quite sure - this probably means it will check node is non null in release.  Might be better to have two verifyex calls one after another.

Reply:
Not sure I understand the concern, but I will change for clarity.